index.html

```diff
@@ -26,9 +26,226 @@
         <nav class="flex-layout">
             <a class="nav-pill active-nav-pill" href="index.html" ><img class="nav-pill-img" src="assets/images/home.png"></a>
             <a class="nav-pill" href="wishlist.html" ><img class="nav-pill-img" src="assets/images/heart.png"></a>
-            <a class="nav-pill" href="cart.html"><img class="nav-pill-img" src="assets/images/bag.png"></a>
+            <a class="nav-pill" href="cart.html"><img class="nav-pill-img" src="assets/images/bag-white.png"></a>
             <a class="nav-pill" href="login.html"><img class="nav-pill-img" src="assets/images/user.png"></a>
         </nav>
     </header>
+
+    <div class="mg-1 flex-column flex-center">
+        <div class="glass-container">
+            <img class="flex-center mg-1 banner-img" src="assets/images/p&b_banner.png">
+        </div>
+        
+        <div class="flex-column flex-center mg-1">
+            <h3 class="mg-tb-1">Why Choose Us ?</h3>
+            <div class="flex-space-btwn flex-center why-us-contanier">
+                <div class="flex-layout flex-center">
+                    <img class="glass-bg" src="assets/images/delivery-truck.png">
+                    <div class="flex-column">
+                        <h5>Free Delivery</h5>
+                        <span class="p-xs">Free deilvery on all orders</span>
+                    </div>
+                </div>
+                <div class="flex-layout flex-center">
+                    <img class="glass-bg" src="assets/images/customer-support.png">
+                    <div class="flex-column">
+                        <h5>24/7 Support</h5>
+                        <span class="p-xs">24 hours customer support</span>
+                    </div>
+                </div>
+                <div class="flex-layout flex-center">
+                    <img class="glass-bg" src="assets/images/exchange.png">
```

Comment:
Samyukta please alt also in your image tags

Reply:
Have added alt to all the images.